### PR TITLE
Updates to yocto setup

### PR DESCRIPTION
**Pull request by mirzak**

### CODE REVIEW

File: 04.Artifacts/01.Yocto-project/01.Building/docs.md [hunk 1]
```diff
@@ -177,6 +168,39 @@ DISTRO_FEATURES_BACKFILL_CONSIDERED = "sysvinit"
...
+#
+# Uncomment below and update the URL to match your configured domain
+# name and provide the path to the generated server.crt file.
+#
+# NOTE! It is recommend that you provide below information in your custom
+# Yocto layer and this is only for demo purposes. See linked documentation
+# for additional information.
+#MENDER_SERVER_URL = "https://docker.mender.io"
+#FILESEXTRAPATHS_prepend_pn-mender := "<DIRECTORY-CONTAINING-server.crt>:"
+#SRC_URI_append_pn-mender = " file://server.crt"
```
Comment (drewmoseley) [inline 04.Artifacts/01.Yocto-project/01.Building/docs.md]: Do we also want to comment that this is only needed for self-signed certs?

Comment (estenberg) [reply 04.Artifacts/01.Yocto-project/01.Building/docs.md]: If we can mention it in a very short/concise way I think. Most people would probably use self-signed I suppose.

Comment (mirzak) [reply 04.Artifacts/01.Yocto-project/01.Building/docs.md]: Pushed an update which mentions this. Good enough?

**Comment by mirzak** [comment]
In regards to back-ports, this should go all they way back to 1.6.x?

**Comment by drewmoseley** [comment]
Yeah, I think it's applicable to all supported branches.

**Pull request merged**